packages/lambda-xyz/src/wmts.capability.ts

```diff
@@ -22,12 +24,12 @@ for (const k of ImageFormatOrder) {
     formats.push(V('Format', 'image/' + k));
 }
 
-const tileMatrixSetId: Partial<Record<TileSetType, string>> = {
-    [TileSetType.aerial]: 'GoogleMapsCompatible',
+const tileMatrixSetId: Partial<Record<string, string>> = {
+    aerial: 'GoogleMapsCompatible',
```

Comment:
Can we do this based on EPSG code rather than name?

Reply:
I had that originally but then I have to convert back and forth and parseInt etc. This is more straight forward

Reply:
Not sure I follow?
```
if( projection == EPSG.Google) return 'GooleMapsCompatible;
throw new Error(`Invalid projection: ${projection}`
```